packages/node_modules/@ciscospark/react-component-adaptive-card/src/index.js

```diff
@@ -1,14 +1,18 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import ReactDOM from 'react-dom';
 import classnames from 'classnames';
-import {getAdaptiveCard} from '@ciscospark/react-component-utils';
+import {getAdaptiveCard, CARD_CONTAINS_IMAGE, replaceIndexWithBlobURL, API_ACTIVITY_VERB} from '@ciscospark/react-component-utils';
 
 class AdaptiveCard extends React.Component {
   constructor(props) {
     super(props);
     this.nodeElement = React.createRef();
     this.state = {
-      childNodes: []
+      childNodes: [],
+      hasReplacedImagesInJSON: false,
+      hasReplacedCard: false,
+      cards: this.props.cards
```

Comment:
Why are you setting a prop as a state?

Reply:
Please Refer:https://github.com/webex/react-widgets/pull/1034/files/5b1a635ef2b1e5eb25e53cf1b45ef0330dcb3df6#r357024069

Reply:
You can store the cards as an instance variables of the class. The states should only be things that render should be "listening to" to update the display. This does not seem the case for `cards`

Reply:
Cards are being consumed by render method to rerender after updating the images into the JSON.